src/steps/tmux.rs

```diff
@@ -23,10 +24,21 @@ use std::os::unix::process::CommandExt as _;
 pub fn run_tpm(ctx: &ExecutionContext) -> Result<()> {
     let tpm = match env::var("TMUX_PLUGIN_MANAGER_PATH") {
         // If `TMUX_PLUGIN_MANAGER_PATH` is set, search for
-        // `$TMUX_PLUGIN_MANAGER_PATH/bin/install_plugins/tpm/bin/update_plugins`
-        Ok(var) => PathBuf::from(var).join("bin/install_plugins/tpm/bin/update_plugins"),
-        // Otherwise, use the default location `~/.tmux/plugins/tpm/bin/update_plugins`
-        Err(_) => HOME_DIR.join(".tmux/plugins/tpm/bin/update_plugins"),
+        // `$TMUX_PLUGIN_MANAGER_PATH/bin/update_plugins`
+        Ok(var) => PathBuf::from(var).join("bin/update_plugins"),
+        Err(_) => {
+            // Otherwise, use the default XDG location `~/.config/tmux/plugins/tpm/bin/update_plugins`
+            // or fallback on the standard default location `~/.tmux/plugins/tpm/bin/update_plugins`
+            #[cfg(unix)]
+            let xdg_path = XDG_DIRS.config_dir().join("tmux/plugins/tpm/bin/update_plugins");
+            #[cfg(windows)]
+            let xdg_path = HOME_DIR.join(".config/tmux/plugins/tpm/bin/update_plugins");
+            if xdg_path.exists() {
+                xdg_path
+            } else {
+                HOME_DIR.join(".tmux/plugins/tpm/bin/update_plugins")
```

Comment:
Can the string `plugins/tpm/bin/update_plugins` be put in a variable?

Reply:
Done
Actually, I introduced 2 constants for standard (sub)paths

Reply:
Perfect, thank you!